README.md

```diff
@@ -127,6 +127,122 @@ When generating configuration, the program parses structured data (JSON) found i
 
 Similarly, the router watches its _own_ `routerConfig` annotations to dynamically construct global Nginx configuration.
 
+## Configuration Guide
+
+### Environment variables
+
+Router configuration is driven almost entirely by annotations on the router's replication controller and the services of all routable applications-- those labeled with `routable=true`.
```

Comment:
should we change this annotation to `deis.io/routable`, or is it fine to leave it without a domain?